index.js

```diff
@@ -45,6 +46,8 @@ import typeof TouchableWithoutFeedback from './Libraries/Components/Touchable/To
 import typeof View from './Libraries/Components/View/View';
 import typeof VirtualizedList from './Libraries/Lists/VirtualizedList';
 import typeof VirtualizedSectionList from './Libraries/Lists/VirtualizedSectionList';
+
+// APIs
```

Comment:
```suggestion
// Modules
```

Reply:
@rickhanlonii ,

Currently on master, in the export section of index.js, there's a section titled API

Would you like us to change this line as well?

https://github.com/facebook/react-native/blob/9c881b0f9712baef7b6dd6eb530bf371b794fade/index.js#L50

Reply:
I see, ok let's keep API!

Reply:
Sure things, thanks :) !